images/runtime/php-fpm/7.2/base.stretch.Dockerfile

```diff
@@ -74,7 +74,7 @@ RUN docker-php-ext-configure pdo_odbc --with-pdo-odbc=unixODBC,/usr \
 #  - https://docs.microsoft.com/en-us/sql/connect/php/installation-tutorial-linux-mac
 #  - https://docs.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-microsoft-odbc-driver-for-sql-server
 RUN set -eux; \
-    if [[ $PHP_VERSION == 7.2.* || $PHP_VERSION == 7.3.* ]]; then \
+    if [[ $PHP_VERSION == 7.3.* ]]; then \
```

Comment:
is this generated from a template? Do weneed this check for 7.3 here if the dockerfile is in the 7.2 folder?

Reply:
Yes it's from the template.base.Dockerfile, we still want to keep it same way as the template.